datastream-kafka/src/main/java/com/linkedin/datastream/kafka/factory/KafkaConsumerFactory.java

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the BSD 2-Clause License. See the LICENSE file in the project root for license information.
  *  See the NOTICE file in the project root for additional information regarding copyright ownership.
  */
-package com.linkedin.datastream.connectors.kafka;
+package com.linkedin.datastream.kafka.factory;
```

Comment:
Just making a note of this to preempt any subsequent dependency validation build failures, if this is a backward incompatible change (it seems to be since this type is changing packages), please make sure to sync this code into the internal `datastream` MP with a major version bump after this code change is merged.